hpOneView/common.py

```diff
@@ -208,6 +208,7 @@ def wrapper(*args, **kwargs):
     # Uncategorized
     # ------------------------------------
     'unmanaged-devices': '/rest/unmanaged-devices',
+    'vc-migration-manager': '/rest/migratable-vc-domains'
```

Comment:
You are correct.  This line is not used currently.  Should I either reference this line in the VcMigrationManager class or remove this line?